pkgs/by-name/un/unfloader/package.nix

```
@@ -0,0 +1,55 @@
+{lib, stdenv, fetchFromGitHub, ncurses, libftdi1, libusb1, pkg-config}:
+
+stdenv.mkDerivation rec {
```

lets uses `finalAttrs` to improve override flexibility:

```suggestion
stdenv.mkDerivation (finalAttrs: {
```
- #315337